demos/continuous_batching/rag/README.md

```diff
@@ -0,0 +1,25 @@
+# RAG demo with all execution steps delegated to the OpenVINO Model Server {#ovms_demos_rag}
+
+
+## Creating models repository for all the endpoints
+
+```bash
+git clone https://github.com/openvinotoolkit/model_server
+cd model_server/demos/common/export_models
+pip install -q -r requirements.txt
+
+mkdir -p models
+python export_model.py text_generation --source_model meta-llama/Meta-Llama-3-8B-Instruct --weight-format int8 --kv_cache_precision u8 --config_file_path models/config_all.json --model_repository_path models 
+python export_model.py embeddings --source_model Alibaba-NLP/gte-large-en-v1.5 --weight-format int8 --config_file_path models/config_all.json
+python export_model.py rerank --source_model BAAI/bge-reranker-large --weight-format int8  --config_file_path models/config_all.json
```

Comment:
I'm not that familiar with the final version of the `export_model.py`, but will setting `--config_file_path models/config_all.json` in multiple consecutive executions expand the config? Just making sure there won't be any overwrite.

Reply:
@mzegla  it will add a model to existing config file